astropy/units/function/core.py

```diff
@@ -193,7 +195,7 @@ def physical_type(self):
         """Return the physical type of the physical unit (e.g., 'length')."""
         return self.physical_unit.physical_type
 
-    def is_equivalent(self, other, equivalencies=[]):
+    def is_equivalent(self, other, equivalencies=None):
```

Comment:
This is wrong - like for regular `is_equivalent`, `None` has the meaning at no equivalencies at all are used.